blog_post/dbt_blog_post.md

```diff
@@ -0,0 +1,163 @@
+interface), or alternatively the REST or gRPC API directly. Currently, PACE applies the policies by
+generating platform-specific SQL statements that create views on the targeted platform, which in
+their turn leverage native access controls to implement the desired access policy. Got a table with
+with sensitive (customer) data? Let PACE create a (privacy-)safe view for it!
+
+[//]: # (Todo: an illustration here?)
+
+After putting PACE out to the public, we started getting similar feedback from people who shared one
+thing: they used [dbt](https://www.getdbt.com/) for all their data processing needs. They were
+wondering how PACE could fit in their picture. Dbt itself, after all, also is an abstraction layer
+on top of databases and data processing platforms. Dbt users define _models_, which are essentially
+SQL statements that translate to tables or views, but with the power of templating, allowing for
+improved structuring and reuse of SQL code. Dbt also functions, to some extent, as a data catalog:
+data owners can specify metadata about their models, and dbt projects inherently provide a
+fairly extensive overview of all available data products, including lineage insights.
+
+So how does PACE fit in this picture? As a server application, PACE stores its data policies in its
+own database, and directly connects to the different processing platforms. Running PACE in a
+dbt-powered data landscape would not make sense, with both tools keeping track of metadata (e.g.
+data policies) and integrating with data platforms (creating tables or views). Therefore, we soon
+concluded to let dbt be responsible for the creation and updating of new relations (PACE's SQL
+views), and reduce the scope of PACE to the generation of platform-specific SQL based on data policy
+yaml specifications. All while minimizing the impact on a dbt user's existing workflow.
+
+To this end, we started by modularizing PACE, extracting the SQL generation into a `core` module,
+and putting everything related to running PACE as a Spring Boot application in a `server` module.
+This opened the way to adding a `dbt` module, which would know how to generate dbt models
+for PACE data policies, packaged and runnable as a simple executable. The dbt module works as
+follows:
+
+1. A user adds PACE field transforms/filters to their existing `schema.yml` files, on model or
+   column level, under dbt's `meta` sections (see the example below).
+2. After a `dbt compile` or `dbt run`, dbt's `target/manifest.json` file is updated as usual,
+   containing the project's complete configuration, including the PACE policy specifications.
+3. The user runs the PACE dbt module (currently a java jar file) from the dbt project's root, which
+   results in the creation of additional model files, each containing the platform-specific SQL that
+   implements the policy.
+4. During the next `dbt run` execution, dbt creates the secure PACE views on the target platform.
```

Comment:
Might be worth making a remark that we have looked extensively at trying to blend PACE more into the execution of the `dbt compile` command, but that dbt in its current state does not offer command line integrations (only SQL hooks).

Reply:
Indeed, good idea.

Reply:
Good one, added it as a footnote.